packages/vehicle-rental-overlay/src/index.js

```diff
@@ -158,7 +158,7 @@ class VehicleRentalOverlay extends MapLayer {
       );
     }
 
-    if (!filteredStations || filteredStations.length === 0) {
+    if (!visible || !filteredStations || filteredStations.length === 0) {
```

Comment:
This works, but we should probably just return null if not visible on line 153. Otherwise, the empty featureGroup might cause confusion to a reader/debugger and we can skip the station filtering above.

Reply:
Makes sense thanks!